delft/textClassification/models.py

```diff
@@ -49,30 +37,30 @@ def getModel(model_config, training_config, load_pretrained_weights=True, local_
 
     # awww Python has no case/switch statement :D
     if (architecture == 'bidLstm_simple'):
-        model = bidLstm_simple(model_config, training_config)
+        model = BidLSTM_simple(model_config, training_config)
     elif (architecture == 'lstm'):
-        model = lstm(model_config, training_config)
+        model = LSTM(model_config, training_config)
     elif (architecture == 'cnn'):
-        model = cnn(model_config, training_config)
+        model = CNN(model_config, training_config)
     elif (architecture == 'cnn2'):
-        model = cnn2(model_config, training_config)
+        model = CNN2(model_config, training_config)
     elif (architecture == 'cnn3'):
-        model = cnn3(model_config, training_config)
+        model = CNN3(model_config, training_config)
     elif (architecture == 'lstm_cnn'):
-        model = lstm_cnn(model_config, training_config)
+        model = LSTM_CNN(model_config, training_config)
     elif (architecture == 'conv'):
-        model = dpcnn(model_config, training_config)
-    elif (architecture == 'mix1'):
-        model = mix1(model_config, training_config)
+        model = DPCNN(model_config, training_config)
```

Comment:
It looks like an error `DPCNN` has its own class/name. We could remove 'conv' as architecture I think.

Reply:
fixed